### DISCUSSION TITLE
newbie confusion

### DISCUSSION THREAD

**Discussion by dtwind62** [turtlebot/turtlebot4#92] [General]
I'm the proud owner of a turtlebot4 standard robot. I bought the robot to learn about robotics and practice my python programming skills.  I'm very new to this and I have a couple of questions. I would like to install the turtlebot4 examples, and depthai . I dont understand which files have to be installed on the turtlebot4 via ssh, and which files need to be installed on my Ubuntu 20.04 laptop, that I also have ros2 Galactic installed on. I was able to run the examples on the IROBOT python playground by switching the create3 wifi switch to bluetooth, and connecting to my android tablet, but I was unable to get web bluetooth working on my ubuntu laptop I was also wondering if there was another way to use the python playground without pulling the top panel off and switching the switch. Thank you in advance for any help I can get to clear things up for me.

**Comment by roni-kreinin** [comment]
@dtwind62 The depthai nodes should already be installed on the RPi4 by default. 

Which examples are you trying to install?

Regarding the iRobot python playground, these questions are specific to the Create 3 and would be better asked [here](https://github.com/iRobotEducation/create3_docs/discussions).

**Reply by dtwind62** [reply]
I found out that the turtlebot4 examples I was trying to installed are already installed on my computer. I'm still new to navigating the ROS2 file system. Thanks a bunch for the help.
